app/assets/styles/pool/create.scss

```diff
@@ -188,7 +185,7 @@ bl-autoscale-formula-picker  {
             display:block;
             padding:0 2px;
             transform:matrix(0.75, 0, 0, 0.75, 0, 0);
```

Comment:
This is style of label in code editor section. Are there any class we can reuse in form? Because I only see <input/> placeholder attribute as a subtitle when blur, I cloned same style for autoscale editor.